chroot-script

```diff
@@ -356,8 +356,13 @@ kernel() {
      if expr "$COMPONENTS" : '.*non-free' >/dev/null ; then
        KERNELPACKAGES="$KERNELPACKAGES firmware-linux"
      fi
-     # shellcheck disable=SC2086
-     DEBIAN_FRONTEND=$DEBIAN_FRONTEND $APTINSTALL $KERNELPACKAGES
+     if [ "$INITRD_GENERATOR" = 'dracut' ] ; then
+       # install dracut here already to avoid needlessly pulling initramfs-tools
+       # shellcheck disable=SC2086
+       DEBIAN_FRONTEND=$DEBIAN_FRONTEND $APTINSTALL $KERNELPACKAGES dracut
+     else
+       DEBIAN_FRONTEND=$DEBIAN_FRONTEND $APTINSTALL $KERNELPACKAGES
+     fi
```

Comment:
IMO it'd be nicer to add dracut to KERNELPACKAGES, or if possible install it before installing the kernel packages. Duplicating the aptinstall call is a bit meh. (also it would need the same shellcheck annotation)

Reply:
Good, let's install dracut together with kernel packages and also remove initramfs-tools from packages file.

Branch updated just now.